### ISSUE TITLE
Delete user account while removing from the account

### ISSUE THREAD

**Issue by pranavrajs** [chatwoot/chatwoot#1925]
If the user is associated with only one account, delete the user record also while removing it from the account.

#### Problem

When a user is removed from an account, we are removing the base user account considering that the user can be present in more than one account. This creates orphan user records in the system if the user is associated with only one account.

#### Solution

Delete user account also if the user has only one account associated with them.

#### Acceptance criteria

- Remove user from the account
  - If the user has only one account associated with them, delete the user record also. 
  - If the user has more than one account, remove the association only.

#### Special considerations

- Removing a user from an account should remove them from the teams, inboxes
- Removing a user from an account should not remove messages sent by the user, instead, it should point to a null value and should be handled in the UI.

**Comment by murtaza-bagwala** [comment]
@pranavrajs shall I pick this up?

**Comment by pranavrajs** [comment]
@murtaza-bagwala Sure. Please let me know if you need anything else from me.

**Comment by murtaza-bagwala** [comment]
@pranavrajs just to confirm currently super_admin can remove the users from the account right?
and we are just removing the association as of now?

**Comment by pranavrajs** [comment]
Yes, if there is only one account associated with that user, we can remove them from the DB.

**Comment by murtaza-bagwala** [comment]
ok, thanks can you assign this ticket to me
